platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/AccountsManager.java

```diff
@@ -53,23 +107,26 @@ private String validateUserId() {
   }
 
   private void validateRotation(String userId) {
-    SharedPreferences sharedPreferences = getSharedPreferences();
-    timestamp = sharedPreferences.getLong(PREFERENCE_TIMESTAMP, 0L);
-    skuToken = sharedPreferences.getString(PREFERENCE_SKU_TOKEN, "");
-    if (timestamp == 0L || TextUtils.isEmpty(skuToken)) {
+    if (TextUtils.isEmpty(skuToken) || timestamp == 0L) {
       skuToken = generateSkuToken(userId);
       timestamp = persistRotation(skuToken);
     }
   }
 
+  @Nullable
```

Comment:
In this case, this method returns non-null.

Reply:
Ohh great catch! Forgot to remove.